src/components/SponsorsSection/Sponsor.jsx

```diff
@@ -8,7 +8,7 @@ import { SponsorContainer, HeadingSection, SponsorIcon, Wrapper } from './styles
 import { sponsors } from '../../../config/content';
 
 const links = {
-  sponsor: 'https://hacknitr21.typeform.com/to/maI5lJ4t',
+  sponsor: 'https://partners.hacknitr.tech/',
 };
```

Comment:
- Revert this change to as it was.
- Use the config object from config/website.js file which contains all the links. 
- Add the community partner link here itself and use it.